package.json

```diff
@@ -50,5 +50,8 @@
     "ghooks": {
       "pre-commit": "npx lint-staged"
     }
-  }
+  },
+  "workspaces": [
```

Comment:
[npm7 workspaces](https://docs.npmjs.com/cli/v7/using-npm/workspaces) support, monorepo symlinking. 

I came across some unresolved dependency issues when running commands inside a package rather than from the root package.

Reply:
did you bootstrap lerna with `npm run bootstrap` ? I never had problems.

Reply:
I did, but must have been user error - works fine when revisited.. will drop.